Project.toml

```
@@ -12,6 +13,7 @@ QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
+CliffordNumbers = "^0.1.2"
```

```suggestion
CliffordNumbers = "0.1.2"
```

The caret is the default, cf. https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers.

---

Will this still allow newer patch versions? My reading of the examples on the Pkg.jl docs was that it seemed more restrictive if you specified patch version.

---

Yes, it will. Both versions (with and without caret) describe the same compat bounds.